wasm/src/lib.rs

```diff
@@ -24,26 +23,47 @@ pub use iroha_wasm_derive::entrypoint;
 pub use parity_scale_codec::{DecodeAll, Encode};
 
 pub mod debug;
-pub mod validator;
 
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-#[no_mangle]
-#[cfg(not(test))]
-#[panic_handler]
-fn panic(_info: &::core::panic::PanicInfo) -> ! {
-    // Need to provide a tiny `panic` implementation for `#![no_std]`.
-    // This translates into an `unreachable` instruction that will
-    // raise a `trap` in the `WebAssembly` if execution of said WASM panics.
-    unreachable!("Program should have aborted")
-}
-
-#[no_mangle]
-#[cfg(not(test))]
-#[alloc_error_handler]
-fn oom(layout: ::core::alloc::Layout) -> ! {
-    panic!("Allocation({} bytes) failed", layout.size())
+/// Initialization macro for programs using `iroha_wasm`.
```

Comment:
We should probably mention in crate-level docs that it's advisable to call this.